### DISCUSSION TITLE
Is Craft CMS open source project? Under what Open Source License does it falls under ?

### DISCUSSION THREAD

**Discussion by Tdnshah** [craftcms/cms#13928] [Q&A]
I see Craft CMS claims to be an open source product, when I was trying to check its licence it doesnt preety much a clear idea of which open source license it falls under like GPL, MPL, MIT Etc..

**Comment by AugustMiller** [comment]
Craft CMS’s source code is public, but it is proprietary software owned by Pixel & Tonic. The full license is available [here](https://github.com/craftcms/cms/blob/develop/LICENSE.md).

We state in [CONTRIBUTING.md](https://github.com/craftcms/cms/blob/develop/CONTRIBUTING.md#craft-isnt-foss) that Craft CMS does _not_ fall under the “FOSS” or “free open source software” umbrella.

**Reply by Tdnshah** [reply]
Oh thanks for the prompt response, so does this mean any commercial use of this software is restricted ?

**Reply by AugustMiller** [reply]
I’m not sure I follow… modification, resale, and redistribution are not allowed (as a discrete software product or embedded in another product that is then licensed to users).

Craft can absolutely be used for commercial projects so long as the edition is properly licensed. In most cases, that means registering with [Craft Console](https://craftcms.com/knowledge-base/what-is-craft-console) and purchasing a Pro license during development or at launch.

Read more about licensing Craft for your project [here](https://craftcms.com/pricing)!
